### ISSUE TITLE
Dockerisation of the project

### ISSUE THREAD

**Issue by ShivanshPlays** [rahulsainlll/git-trace#30]
I can work on dockerisation of this entire project.

**Comment by Aarzookhunger** [comment]
@ShivanshPlays someone else also raised the same issue, so do you preferably want to work on this or shall I assign him?

**Comment by ShivanshPlays** [comment]
Yes, I want to work on this issue, but as discussed with @rahulsainlll this issue shall be addressed during the later half of the program.
Although I would be more than happy to work on it, whenever it is assigned to me

**Comment by rahulsainlll** [comment]
> @ShivanshPlays someone else also raised the same issue, so do you preferably want to work on this or shall I assign him?

it will be good if we do this after we merge other PRs first it can be done 1-2 week before gssoc end do remind me @ShivanshPlays before it ends i'll assign with other this under level3

**Comment by ShivanshPlays** [comment]
Ok , I'll do it in the later half of the program

**Comment by unknown91tech** [comment]
@ShivanshPlays are you working on this issue?

**Comment by ShivanshPlays** [comment]
@unknown91tech bro, the project admin has told me to work on this in the last week

**Comment by unknown91tech** [comment]
@ShivanshPlays okay no problem
